### PR TITLE
Update Ubuntu LTS and add Git to the Dockerfile

### PR DESCRIPTION
**Pull request by soerenmartius**
# Reason for This Change

Our Dockerfile is based on an outdated version of Ubuntu and missed Git as a dependency.

## Description of Changes

Updates ubuntu to current LTS `22.04` and adds Git as a dependency.

### CODE REVIEW

**Review by MordecaiMalignatus — APPROVED** [review]

**Review by MordecaiMalignatus — APPROVED** [review]
It might be a good idea to clarify what exactly the image is *for*, or to disambiguate use-cases for the docker image. 

Is it for running in CI? In that case, it needs more stuff to be able to do its job. 
Is it for people not wanting to run/download an untrusted binary? In this case advising people to mount things into it is fine, but that also needs to be specified, maybe even to the point of having two images.

File: docs/installation.md [hunk 1]
```diff
@@ -77,6 +77,18 @@ docker run ghcr.io/terramate-io/terramate
 We also provide container images tagged with specific release versions.
 To view a list of available container image tags, visit this [link](https://github.com/terramate-io/terramate/pkgs/container/terramate/versions).
 
+**Note:** Our image doesn't come with additional dependencies such as Terraform. We recommend building
+your own image by using our image as the base image to install additional dependencies required.
+
+We recommend mounting your Terramate project as a Docker mount volume to use it inside the container. Please note
```
Comment (i4ki) [inline docs/installation.md:83]:
```suggestion
We recommend mounting your Terramate project as a Docker mount volume to use it inside the container. Please note:
```

**Review by i4ki — DISMISSED** [review]
lgtm

File: docs/installation.md (hunk 1 above)
Comment (soerenmartius) [reply docs/installation.md]: thanks! I rephrased the sentence

**Review by MordecaiMalignatus — APPROVED** [review]

**Pull request merged**